#!/usr/bin/env python
PACKAGE = "daho_vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#Color filter
gen.add("minH", int_t, 0, "minH Color Filter Lapangan", 30, 0, 255)
gen.add("minS", int_t, 0, "minS Color Filter Lapangan", 63, 0, 255)
gen.add("minV", int_t, 0, "minV Color Filter Lapangan", 87, 0, 255)
gen.add("maxH", int_t, 0, "maxH Color Filter Lapangan", 54, 0, 255)
gen.add("maxS", int_t, 0, "maxS Color Filter Lapangan", 185, 0, 255)
gen.add("maxV", int_t, 0, "maxV Color Filter Lapangan", 196, 0, 255)

exit(gen.generate(PACKAGE, "daho_vision", "Field"))